README.md

```diff
@@ -43,7 +44,8 @@ Our team will be creating a website to help people find recipes for maintaining
 - Users will be able to set their own preferences for food recipes; such as the level of spiciness, the food cultures, and many more.
 - The system will save the information and organize the pages to fit the user preference.
 
-4. Users will be able to filter food recipes by:
+4. Filter
+- Users will be able to filter food recipes by:
     a. Popularity
```

Comment:
Can we make these bullet points? Using a, b, c doesn't show up correctly in markdown. This is how it looks: https://github.com/CS157A-Team18/CS157A-Team18/blob/9a62bbbeefb70266ae382699be7079cdad71df9f/README.md

Reply:
Yes. I will make the revision.